web/docs/language/features.md

```diff
@@ -178,138 +178,246 @@ Currently entities can be accessed only in Operations (Queries & Actions), so ch
 
 ## Queries and Actions (aka Operations)
 
-In Wasp, main interaction between client and server happens via Operations, of which two types exist: Queries and Actions.
+In Wasp, the client and the server interact with each other through Operations.
+Wasp currently supports two kinds of Operations: **Queries** and **Actions**.
 
 ### Query
 
-Queries are NodeJS functions that don't modify any state. Normally they fetch certain resources, process them and return result. They are executed on server. 
+Queries are resource-fetching functions that don't modify the state. Each Query should fetch, process, and return a particular resource.
 
-To create a Wasp Query, we need two things: declaration in Wasp and implementation in NodeJS:
+Queries are implemented in NodeJS and executed within the server's context.
+However, Wasp will generate the code that lets you call the Query from any point in your code (i.e., client, server) using the same interface.
```

Comment:
Starting a sentence with "However," is not really very idiomatic English. Prefer "But" but I'd just leave it out here as what's the contradiction really? In general it's a good idea to avoid the word "will" in sentences; you can often do so and the sentence becomes simpler

"Wasp generates code that lets you call the Query from anywhere in your code (client or server) using the same interface."

(I also got rid of the `i.e.` as client and server are the only two example where you can do it; we have no idea yet whether the upcoming jobs let you use queries).

Reply:
I'll drop the however. Don't know why it's there, could be a leftover from one of my previous iterations of the text.

I can definitely remove the `i.e.` for cosmetic purposes ("client or server" looks much nicer than "i.e., client, server"), but I think [the two options are equivalent](https://www.merriam-webster.com/words-at-play/ie-vs-eg-abbreviation-meaning-usage-difference) from a semantic standpoint.